x/crosschain/types/constant.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+const (
+	// CCTXIndexLength is the length of a crosschain transaction index
+	CCTXIndexLength          = 66
+	MaxOutboundTrackerHashes = 5
```

Comment:
Not related to this PR directly but might be great to add a comment for this one in the meantime